changelog/12344.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Could you please remove this changelog ? We typically do not do changelogs from docs only changes

Reply:
I removed this file :+1: